crates/kitsune_p2p/bootstrap/CHANGELOG.md

```diff
@@ -1,6 +1,5 @@
 ---
-semver_increment_mode: patch
```

Comment:
@steveej Shouldn't this have been deleted by the last release?

Reply:
No, there wasn't AFAICS https://github.com/holochain/holochain/pulls?q=is%3Apr+is%3Aopen+back+into+develop-0.1